pkg/diffs/diffs.go

```diff
@@ -72,6 +73,25 @@ func GetChangedCiopConfigs(masterConfig, prConfig config.DataByFilename, logger
 			}
 		}
 
+		oldResources := oldConfig.Configuration.Resources
+		newResources := newConfig.Configuration.Resources
+
+		for as, resource := range newResources {
+			if as == "*" && !equality.Semantic.DeepEqual(oldResources["*"], resource) {
+				for testName := range newTests {
+					jobs.Insert(testName)
+					ret[filename] = newConfig
+				}
+				break
+			}
+
+			if !equality.Semantic.DeepEqual(oldResources[as], resource) {
+				logger.WithField(logCiopConfig, filename).Info(changedCiopConfigMsg)
+				ret[filename] = newConfig
+				jobs.Insert(as)
```

Comment:
Why can you insert the resource name identifier (could be a container name, build name, etc) as a job>

Reply:
Wait, I thought that the resource name is the same as the test name.

Reply:
Not necessarily - the resource name could be `*` or a build name, etc

Reply:
(we have logic to manage the matching)